code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -188,251 +194,3 @@
 			user.updateappearance(mutcolor_update=1)
 			user.domutcheck()
 			user << "You assume the face of [H]. Are you satisfied?"
-
-///Ash Walkers
-
-/mob/living/simple_animal/hostile/spawner/ash_walker
-	name = "ash walker nest"
-	desc = "A nest built around a necropolis tendril. The eggs seem to grow unnaturally fast..."
-	icon = 'icons/mob/nest.dmi'
-	icon_state = "ash_walker_nest"
-	icon_living = "ash_walker_nest"
-	faction = list("ashwalker")
-	health = 200
-	maxHealth = 200
-	loot = list(/obj/effect/gibspawner, /obj/item/device/assembly/signaler/anomaly)
-	del_on_death = 1
-	var/meat_counter
-
-/mob/living/simple_animal/hostile/spawner/ash_walker/Life()
-	..()
-	if(!stat)
-		consume()
-		spawn_mob()
-
-/mob/living/simple_animal/hostile/spawner/ash_walker/proc/consume()
-	for(var/mob/living/H in view(src,1)) //Only for corpse right next to/on same tile
-		if(H.stat)
-			visible_message("<span class='warning'>Tendrils reach out from \the [src.name] pulling [H] in! Blood seeps over the eggs as [H] is devoured.</span>")
-			playsound(get_turf(src),'sound/magic/Demon_consume.ogg', 100, 1)
-			if(istype(H,/mob/living/simple_animal/hostile/megafauna/dragon))
-				meat_counter += 20
-			else
-				meat_counter ++
-			for(var/obj/item/W in H)
-				H.unEquip(W)
-			H.gib()
-
-/mob/living/simple_animal/hostile/spawner/ash_walker/spawn_mob()
-	if(meat_counter >= 2)
-		new /obj/effect/mob_spawn/human/ash_walker(get_step(src.loc, SOUTH))
-		visible_message("<span class='danger'>An egg is ready to hatch!</span>")
-		meat_counter -= 2
-
```

Comment:
Everything above in this particular .dm looks important and i don't think the ash walker tendril is defined elsewhere